### ISSUE TITLE
No way to print it double sided

### ISSUE THREAD

**Issue by amiller** [ryepdx/ethaddress.org#23]
If you look at bitcoinpaperwallet.com, the wallet front is aligned to take up exactly half a page, so if you print it once, flip it over, then print another, the back matches up.

**Comment by dankinchen** [comment]
yeah, I cannot get this to print 2 sided either. I was able to find a way only in landscape, but the flaps did not match. It clearly is not right.
